### ISSUE TITLE
"What is" paragraphs on each feature

### ISSUE THREAD

**Issue by pkbullock** [SharePoint/sp-usage-docs#22]
#### Article suggestion

This is something that crops up a lot in documentation, a brief description say no more than 1-2 paragraphs on What is SharePoint, What are Content Types, What is....X

I have built a collection of mini summaries, that i put in documentation to explain in the simplest terms what that feature is, its quite often i explain to different clients what X feature does in solutions.

**Comment by pkbullock** [comment]
Happy for you to assign to me.

**Comment by pkbullock** [comment]
I'm going to close this as seems this is covered by that, thanks Marc!

**Comment by sympmarc** [comment]
I'm also thinking that we have a sort of building block approach here:
* Glossary terms which can be referred to in...
* "What is" articles about a single topic in more depth, which can be referred to in...
* Making decisions articles which enable us to decide how to do things more effectively.

**Comment by pkbullock** [comment]
Yeah i see your point, lets reopen this, to keep this on the radar. Tbh lots to add to glossary - difficult to think what terminology to add, when we use it daily we don't always realize others do not know.

**Comment by pkbullock** [comment]
@sympmarc is this worth keeping open, feels like a continuous activity?

**Comment by sympmarc** [comment]
It is, but I suppose it could stay open forever. It feels like we’ve gotten somewhere with the glossary, but don’t have as much in the Basics session to cover the goals here.
